pkg/integration/dockerCmd_utils_test.go

```diff
@@ -389,7 +387,7 @@ func TestHelperProcess(t *testing.T) {
 			fmt.Fprintf(os.Stderr, "an error has occurred")
 			os.Exit(1)
 		case "a command that times out":
-			time.Sleep(10 * time.Millisecond)
+			time.Sleep(30 * time.Millisecond)
```

Comment:
Is this change related to this PR?

I'm not familiar with this func, but what is this 'case' doing? It seems like this isn't much different than the 'echo hello' case below, especially since I wouldn't have expected a process that was killed due to a timeout to exit with non-zero.